### ISSUE TITLE
nn.DataParallel doesn't work

### ISSUE THREAD

**Issue by Colin97** [facebookresearch/SparseConvNet#148]
self.sparseModel = scn.Sequential().add(
           scn.InputLayer(self.dimension, torch.LongTensor([self.resolution * 8 + 15] * 3), mode=3)).add(
           scn.SubmanifoldConvolution(self.dimension, 1, self.m, 3, False)).add(
           scn.FullyConvolutionalNet(self.dimension, self.reps, self.nPlanes, residual_blocks=False, downsample=[3,2])).add(
           scn.BatchNormReLU(sum(self.nPlanes))).add(
           scn.OutputLayer(self.dimension))

when I use only one GPU to run the model with a bath_size=34, it takes ~3s for each iteration.
However, when I use 8 GPUs with a bath_size = 272, it takes ~25s for each iteration.
It seems that nn.DataParallel(model) doesn't work at all.

**Comment by btgraham** [comment]
Sorry, only single GPU training is supported for now.
